prisma/schema.prisma

```diff
@@ -52,19 +52,19 @@ model Session {
 }
 
 model User {
-    id              String     @id @default(cuid())
-    name            String?
-    email           String?    @unique
-    emailVerified   DateTime?
-    image           String?
-    role            Role       @default(READER)
-    accounts        Account[]
-    sessions        Session[]
-    blogPost        BlogPost[]
-    comment         Comment[]
-    createdAt       DateTime  @default(now()) @db.Timestamptz(3) // with a timezone
+    id            String     @id @default(cuid())
+    name          String?
+    email         String?    @unique
+    emailVerified DateTime?
+    image         String?
+    role          Role       @default(READER)
+    accounts      Account[]
+    sessions      Session[]
+    blogPosts     BlogPost[]
+    comments      Comment[]
```

Comment:
Are we going to need to do a migration for these changes?

Reply:
I don't think so, it worked fine locally. I didn't have to run the migration script after that change. You can pull and test to confirm